From 37c11b5b83ccc184e10bdea24dc4c8ed02474d1d Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 21 Jan 2025 17:49:35 +0800
Subject: [PATCH 5/8] loonggpu: gsgpu_dc_connector: gsgpu-bridge: drop an
 unused variable i

A variable `i' was defined under multiple functions, triggering
`-Wunused-variable'.

Drop unused variable to fix compiler warnings.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 gsgpu-bridge/bridge_phy.c  | 1 -
 gsgpu/gsgpu_dc_connector.c | 1 -
 2 files changed, 2 deletions(-)

diff --git a/gsgpu-bridge/bridge_phy.c b/gsgpu-bridge/bridge_phy.c
index f67d8f3..921583e 100644
--- a/gsgpu-bridge/bridge_phy.c
+++ b/gsgpu-bridge/bridge_phy.c
@@ -66,7 +66,6 @@ static struct drm_encoder *
 bridge_phy_connector_best_encoder(struct drm_connector *connector)
 {
 	struct drm_encoder *encoder;
-	int i;
 
 	/* pick the first one */
 	lg_drm_connector_for_each_possible_encoder(connector, encoder, i)
diff --git a/gsgpu/gsgpu_dc_connector.c b/gsgpu/gsgpu_dc_connector.c
index 86daef6..b0a9185 100644
--- a/gsgpu/gsgpu_dc_connector.c
+++ b/gsgpu/gsgpu_dc_connector.c
@@ -14,7 +14,6 @@
 
 static struct drm_encoder *best_encoder(struct drm_connector *connector)
 {
-	int i;
 	struct drm_encoder *encoder;
 
 	/* pick the first one */
-- 
2.48.1

